XamlControlsGallery/ControlPages/InfoBadgePage.xaml

```diff
@@ -0,0 +1,156 @@
+
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="Style" Value="{x:Bind InfoBadgeStyleComboBox.SelectedValue, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example3" HeaderText="Placing an InfoBadge Inside Another Control" HorizontalContentAlignment="Stretch" >
+            <local:ControlExample.Example>
+                <Button Padding="0" Width="200" Height="60" HorizontalAlignment="Center" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch">
+                    <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Width="Auto" Height="Auto">
+                        <SymbolIcon Symbol="Sync" HorizontalAlignment="Center" />
+                        <muxc:InfoBadge Background="#C42B1C" HorizontalAlignment="Right" VerticalAlignment="Top">
+                            <muxc:InfoBadge.IconSource>
+                                <muxc:FontIconSource FontFamily="Segoe MDL2 Assets" Glyph="&#xF13C;"/>
+                            </muxc:InfoBadge.IconSource>
+                        </muxc:InfoBadge>
+                    </Grid>
+                </Button>
+
+            </local:ControlExample.Example>
+
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;Button Padding="0" Width="200" Height="60" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch"&gt;
+    &lt;Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Width="Auto" Height="Auto"&gt;
+        &lt;SymbolIcon Symbol="Sync" HorizontalAlignment="Center"InfoBadgeStyle}" /&gt;
+        &lt;muxc:InfoBadge Background="#C42B1C" HorizontalAlignment="Right" VerticalAlignment="Top"/&gt;
+            &lt;muxc:InfoBadge.IconSource&gt;
+                &lt;muxc:FontIconSource FontFamily="Segoe MDL2 Assets" Glyph="&#xF13C;" /&gt;
+            &lt;/muxc:InfoBadge.IconSource&gt;
+        &lt;/muxc:InfoBadge&gt;
+    &lt;/Grid&gt;
+&lt;/Button&gt;                    
+                </x:String>
+            </local:ControlExample.Xaml>
+
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="Style" Value="{x:Bind InfoBadgeStyleComboBox.SelectedValue, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example4" HeaderText="InfoBadge with Dynamic Value" HorizontalContentAlignment="Stretch" >
+            <local:ControlExample.Example>
+                <muxc:InfoBadge x:Name="DynamicInfoBadge"
+                                HorizontalAlignment="Center"
+                                Value="{Binding ElementName=ValueNumberBox, Path=Value, Mode=TwoWay}" />
+            </local:ControlExample.Example>
+
+            <local:ControlExample.Options>
+                <StackPanel Width="130">
```

Comment:
We should probably increase this too, entering text is a bit hard:
![Screnshot of truncated text in numberbox](https://user-images.githubusercontent.com/16122379/132555257-45fc6703-9fda-4d4a-93b3-4914fca038ba.png)